### PR TITLE
refactor: extract features to separate json file

### PR DESCRIPTION
**Pull request by vardan-arm**

### CODE REVIEW

**Review by karolsojko — APPROVED** [review]

**Comment by karolsojko** [comment]
@vardan-arm you need to bump the package version

**Comment by vardan-arm** [comment]
> @vardan-arm you need to bump the package version

Ah, forgot it again, thanks 👍

**Comment by vardan-arm** [comment]
@mo @karolsojko @antsgar if you are ok with these changes, can we merge this PR, so that I'll put this new version to other affected repos as well?

File: packages/features/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/features",
-  "version": "1.2.3",
+  "version": "1.2.4",
```
Comment (karolsojko) [inline packages/features/package.json:3]: can we change this to 1.3.0?

@vardan-arm feel free to merge since the PR is approved

**Review by karolsojko — APPROVED** [review]

File: packages/features/package.json (hunk 1 above)
Comment (vardan-arm) [reply packages/features/package.json]: Done. And got it)

**Pull request merged**